doc/Type/UInt.pod6

```diff
@@ -0,0 +1,12 @@
+=begin pod
+
+=TITLE class UInt
```

Comment:
This probably should be `subset` not `class`. You can't instantiate a subset, for example, but you can a class.